app/3.0/toolbox/localize.md

```diff
@@ -4,113 +4,123 @@ title: Localization
+class AppLocalize extends mixinBehaviors([AppLocalizeBehavior], PolymerElement) {
+```
+  
 ## Add localization to your app
 
 The main application is usually responsible for loading the localized messages and setting the
 current language.
 
-Sample application (class style elements) {.caption}
-
-```html
-<dom-module id="x-app">
-  <template>
-    <!-- use the localize method to localize text -->
-    <div>{{localize('hello', 'name', 'Batman')}}</div>
-  </template>
-  <script>
-    class XApp extends Polymer.mixinBehaviors([Polymer.AppLocalizeBehavior], Polymer.Element) {
-      static get is() { return 'x-app'}
-
-      static get properties() {
-        return {
-          // set the current language—shared across all elements in the app
-          // that use AppLocalizeBehavior
-          language: {
-            value: 'en'
-          },
-
-          // Initialize locale data
-          resources: {
-            value() {
-              return {
-                'en': { 'hello': 'My name is {name}.' },
-                'fr': { 'hello': 'Je m\'apelle {name}.' }
-              }
-            }
-          }
-        };
+app-localize.js {.caption}
+
+```js
+// Imports go here
+class AppLocalize extends mixinBehaviors([AppLocalizeBehavior], PolymerElement) {
+  static get properties(){
+    return{
+      language: {
+        value: 'en'
+      },
+      resources: {
+        value: function() {
+          return {
+            'en': { 'hello': 'My name is {name}.' },
+            'fr': { 'hello': 'Je m\'apelle {name}.' }
+          };
+        }
       }
-    }
-
-    customElements.define(XApp.is, XApp);
-  </script>
-</dom-module>
+    };
+  }
+  static get template(){
+    return html`
+      <div>{{localize('hello', 'name', 'Batman')}}</div>
+    `;
+  }
+}
+customElements.define('app-localize', AppLocalize);
 ```
 
-More typically, the app loads resources from an external file, as shown in the next example.
-
-Sample application (hybrid elements) {.caption}
+index.html {.caption}
 
 ```html
-<dom-module id="x-app">
-   <template>
-    <!-- use the localize method to localize text -->
-    <div>{{localize('hello', 'name', 'Batman')}}</div>
-   </template>
-   <script>
-      Polymer({
-        is: "x-app",
-
-        // include the behavior
-        behaviors: [
-          Polymer.AppLocalizeBehavior
-        ],
-
-        // set the current language—shared across all elements in the app
-        // that use AppLocalizeBehavior
-        properties: {
-          language: {
-            value: 'en'
-          },
-        }
-
-        // load localized messages
-        attached: function() {
-          this.loadResources(this.resolveUrl('locales.json'));
-        },
-      });
-   </script>
-</dom-module>
+<head>
+  <script src="./node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
+  <script type="module" src="app-localize.js"></script>
```

Comment:
again, if we could change the name to something like localized=app it would be clearer what you'd importing.